src/controllers/user.controller.js

```diff
@@ -0,0 +1,90 @@
+const userService = require('./../services/user.service');
+
+const getUsers = async (req, res) => {
+  try {
+    const users = await userService.getAllUsers();
+
+    res.send(users);
+  } catch (error) {
+    res.status(500).send('Server error');
+  }
+};
+
+const getOneUser = async (req, res) => {
+  const { id } = req.params;
+
+  try {
+    const user = await userService.getUserById(id);
+
+    if (!user) {
+      return res.status(404).send('User not found');
+    }
+    res.send(user);
+  } catch (error) {
+    res.status(500).send('Server error');
+  }
+};
+
+const createUser = async (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    return res.sendStatus(400);
```

Comment:
Returning a 400 status code without a message might be unclear to the client. Consider sending a message like 'Name is required' to clarify the reason for the bad request.